packages/core/src/components/Tooltip/index.tsx

```diff
@@ -150,12 +151,15 @@ export class Tooltip extends React.Component<Props & WithStylesProps, State> {
   };
 
   private handleEnter = () => {
-    const { current } = this.containerRef;
+    /* istanbul ignore next: refs are hard */
+    this.rafHandle = requestAnimationFrame(() => {
+      const { current } = this.containerRef;
 
-    /* istanbul ignore if: refs are hard */
-    if (current) {
-      this.setState({ targetRect: current.getBoundingClientRect() });
-    }
+      /* istanbul ignore if: refs are hard */
+      if (current) {
```

Comment:
Probably need `current && this.mounted`.

Reply:
we cancel this callback when we set `this.mounted=false`, but probably worth it to add. updated.